src/appshell/view/preferences/generalpreferencesmodel.cpp

```diff
@@ -35,10 +35,12 @@ void GeneralPreferencesModel::load()
 {
     languagesConfiguration()->currentLanguageCode().ch.onReceive(this, [this](const QString& languageCode) {
         emit currentLanguageCodeChanged(languageCode);
+        sesIsNeedRestart(true);
     });
 
     languagesService()->languages().ch.onReceive(this, [this](const LanguagesHash&) {
         emit languagesChanged(languages());
+        sesIsNeedRestart(true);
```

Comment:
I expected that the notification would be sent only if the language changed, but it seems to be always sent. I hope it doesn't matter that now any language update check will require us to restart the application.